### ISSUE TITLE
spell_group_stack_rules && spell_group

### ISSUE THREAD

**Issue by Kitzunu** [azerothcore/azerothcore-wotlk#5578]
## CURRENT BEHAVIOUR:

Currently we use sunwells system of flags, stack_rule, and special_flags

I don't see any point of this as we have some of issues concerning just stacking of spells, a lot of the sunwell created ones nobody knows what they do.

My proposal is that we just import TCs system that we know is working and can fix the issues we have

Basically I think we should just import TCs system that we know works correctly

**Comment by FrancescoBorzi** [comment]
I don't know about spell flags but I like your big commit hash

**Comment by Kitzunu** [comment]
Hehe.

I will put this on my radar for the summer unless somone else picks it up

**Comment by Footman** [comment]
you are right. i see no reason to keep this system. from code tcs looks much better
